examples/golang-hello-vendor/Makefile

```diff
@@ -26,8 +26,8 @@ $(SRCZIP): $(SRCS) $(VENDORS)
 	cd src ; zip ../$(SRCZIP) -r *
 
 clean:
-	wsk action delete $(PACKAGE)/$(NAME)
-	rm  $(BINZIP) $(SRCZIP) package.done
+	-$(WSK)action delete $(PACKAGE)/$(NAME)
```

Comment:
Is this prefix `-` is need it?

Reply:
Yes, it ignore errors,  otherwise if you do not  have the actions the main makefile will break at the first run (make clean devel test)